packages/server/index.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+const cors = require('cors');
+const { ethers } = require("ethers");
+
+const { upload } = require('./utils');
+const Config = require("./config.json");
+const Secrets = require("./secrets.json");
+
+const port = Config.server.port;
+
+const app = express()
+
+app.use(cors());
+app.use(express.json({ limit: '50mb' }));
```

Comment:
sending base64 encoded image resulted in limit error

Reply:
Yes, we decided not to use base64 encoded images https://kernel-community.slack.com/archives/C031ES44SA3/p1652523840528699?thread_ts=1652483005.662999&cid=C031ES44SA3